pkg/plugins/resources/shell/environment.go

```diff
@@ -0,0 +1,45 @@
+package shell
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/sirupsen/logrus"
+)
+
+const (
+	// DryRunVariableName specifies the environment variable used within shell script to detect if we are in dryrun mode
+	DryRunVariableName = "DRY_RUN"
+)
+
+type Environment struct {
+	// Name defines the environment variable name
+	Name string `yaml:",omitempty"`
+	// Value defines the environment variable value
+	Value string `yaml:",omitempty"`
+}
+
+func (e *Environment) String() string {
```

Comment:
```suggestion
func (e Environment) String() string {
```


This method does not change the receiver